virtualsmartcard/src/vpicc/virtualsmartcard/tests/CardGenerator_test.py

```diff
@@ -51,7 +54,36 @@ def test_load_nPA_from_file_nPA_from_file(self):
 
     def test_load_nonexistent_file(self):
         with self.assertRaises(anydbm.error):
-            self.nPA_generator.loadCard(self.filename)
+            self.card_generator.loadCard(self.filename)
+
+    def test_get_and_set_card(self):
+        self.card_generator.generateCard()
+        mf, sam = self.card_generator.getCard()
+        local_generator= CardGenerator(self.card_type)
+        local_generator.setCard(mf, sam)
+
+class TestNPACardGenerator(ISO7816GeneratorTest):
+
+    card_type = 'nPA'
+
+    def test_readDatagroups(self):
+        path = os.path.dirname(__file__)
+        datagroupsFile = path + "/Example_Dataset_Mueller_Gertrud.txt"
```

Comment:
A relative path to `../../../../npa-example-data/Example_Dataset_Mueller_Gertrud.txt` would eliminate the need for the symlink. This way the tests could also be used on Windows, right?